src/main/java/com/starbucks/model/Order.java

```diff
@@ -1,81 +1,50 @@
 package com.starbucks.model;
 
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-import java.util.UUID;
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
 
+import javax.persistence.*;
+import java.util.UUID;
 
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+@Setter
+@Getter
+@Builder
+@Table(name = "Order", schema = "starbucks", catalog = "postgres")
 public class Order {
 
     @Id
-    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "seq")
+    @Column(name = "id", nullable = false)
+    @GeneratedValue(generator = "UUID")
+    @GenericGenerator(name = "UUID", strategy = "org.hibernate.id.UUIDGenerator")
     private UUID id;
 
+    @Column(name = "time", nullable = false)
     private Integer time;
+    @Column(name = "beverages", nullable = false)
     private String beverages;
+    @Column(name = "size", nullable = false)
     private String size;
+    @Column(name = "sugar", nullable = false)
     private Integer sugar;
 
-    public Order(Integer time , String beverage , String size , Integer sugar) {
-        this.id = UUID.randomUUID ( );
-        this.time = time;
-        this.beverages = beverage;
-        this.size = size;
-        this.sugar = sugar;
-    }
-
-    public Order( ) {
-    }
-
-    public Integer getTime( ) {
-        return time;
-    }
-
-    public void setTime(Integer time) {
-        this.time = time;
-    }
 
-    public UUID getId( ) {
-        return id;
-    }
-
-    public void setId(UUID id) {
-        this.id = id;
-    }
+    @Enumerated(EnumType.STRING)
+    @Column(name = "statusOrder", nullable = false)
+    private Status status;
 
-    public String getBeverages( ) {
-        return beverages;
-    }
-
-    public void setBeverages(String beverages) {
-        this.beverages = beverages;
-    }
-
-    public String getSize( ) {
-        return size;
-    }
-
-    public void setSize(String size) {
-        this.size = size;
-    }
-
-    public Integer getSugar( ) {
-        return sugar;
-    }
-
-    public void setSugar(Integer sugar) {
-        this.sugar = sugar;
-    }
 
     @Override
     public String toString( ) {
```

Comment:
Isn't Lombok annotation @ToString doesn't do it?

Reply:
Yes